eng/automation/changelog/src/test/java/com/azure/resourcemanager/tools/changelog/BreakingChangeTests.java

```diff
@@ -0,0 +1,54 @@
+package com.azure.resourcemanager.tools.changelog;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.resourcemanager.tools.changelog.utils.BreakingChange;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.net.URL;
+
+public class BreakingChangeTests {
+    @Test
+    public void testBreakingChange() {
+        BreakingChange breakingChange = BreakingChange.onJavaClass("MyClass");
+        breakingChange.setClassLevelChangeType(BreakingChange.Type.REMOVED);
+        Assertions.assertEquals("#### `MyClass` was removed\n\n", breakingChange.getForChangelog());
+        Assertions.assertEquals("Class `MyClass` was removed.", breakingChange.getItems().iterator().next());
+
+        String methodLevelContent = "`sku()` was removed";
+        String methodLevelContent2 = "`tier()` was removed";
+        breakingChange.addMethodLevelChange(methodLevelContent);
+        breakingChange.addMethodLevelChange(methodLevelContent2);
+        // test deduplicate
+        breakingChange.addMethodLevelChange(methodLevelContent2);
+
+        Assertions.assertEquals(2, breakingChange.getItems().size());
+        Assertions.assertTrue(breakingChange.getForChangelog().contains("#### `MyClass` was modified"));
```

Comment:
#### `MyClass` was modified --> #### `MyClass` was modified\n\n
?

Reply:
This is a test to prevent regression for existing changelog.md: https://github.com/azure-sdk/azure-sdk-for-java/pull/13130/files#diff-c6b42203b8bd8e0866aaec71668d3da98ddb32e1b8653407b269b8eee7e4788b

Not related to breakingChangeItems.